scripts/release/mule/package/deb/package.sh

```diff
@@ -7,14 +7,17 @@ echo
 date "+build_release begin PACKAGE DEB stage %Y%m%d_%H%M%S"
 echo
 
-BRANCH=${BRANCH:-$(./scripts/compute_branch.sh)}
-CHANNEL=${CHANNEL:-$(./scripts/compute_branch_channel.sh "$BRANCH")}
+if [ -z "$NETWORK" ]; then
+    echo "[$0] NETWORK=$NETWORK is missing."
```

Comment:
Won't this always be `NETWORK= is missing.`?

Reply:
Ha, yeah, I was just looking at this above ^^ and thinking "why the hell did I do that?"

Will change :)